test/auro-flightline.test.js

```diff
@@ -13,6 +13,15 @@ describe('auro-flightline', () => {
     await expect(el).to.be.accessible();
   });
 
+  it('auro-flightline with canceled status', async () => {
+    const el = await fixture(html`
+      <auro-flightline canceled="true">
```

Comment:
In HTML a boolean value is a value that equals the conical string of itself. E.g. `canceled="canceled"` is true. While your code will resolve as `true` because `canceled=" "` will also equal true, the reverse `canceled="false"` will not not resolve as `false`, but `true. 

```suggestion
      <auro-flightline canceled>
```